cmd/occollector/app/collector/processors.go

```diff
@@ -97,11 +83,24 @@ func buildQueuedSpanProcessor(
 			sender.HTTPTimeout(thriftHTTPSenderOpts.Timeout),
 		)
 	}
+	doneFns, traceExporters, _ := createExporters(opts.RawConfig, logger)
 
-	if spanSender == nil {
-		logger.Fatal("Unrecognized sender type or no exporters configured", zap.String("SenderType", string(opts.SenderType)))
+	if spanSender == nil && len(traceExporters) == 0 {
+		logger.Fatal("Unrecognized sender type or no exporters configured")
 	}
 
+	allSendersAndExporters := make([]processor.SpanProcessor, 0, 2)
+	if spanSender != nil {
+		allSendersAndExporters = append(allSendersAndExporters, spanSender)
+	}
+	if len(traceExporters) > 0 {
+		allSendersAndExporters = append(
+			allSendersAndExporters, processor.NewTraceExporterProcessor(traceExporters...),
+		)
+	}
+
+	sender := processor.NewMultiSpanProcessor(allSendersAndExporters)
```

Comment:
I think we need one instance of the queue per destination instead, otherwise failure to one destination will cause retry of the same data on all destinations.